jpa/org.eclipse.persistence.jpa/src/main/java/org/eclipse/persistence/internal/jpa/weaving/ClassWeaver.java

```diff
@@ -216,7 +219,9 @@ public static boolean isJAXBOnPath() {
     }
 
     public ClassWeaver(ClassVisitor classWriter, ClassDetails classDetails) {
-        super(classWriter);
+        super(Opcodes.valueInt("ASM9"), classWriter);
```

Comment:
`Opcodes.valueInt(..)` should probably correspond to the version of SE the application is running on